Add known limitations for VSCE (#31019)
sourcegraph/sourcegraph: doc/admin/how-to/troubleshoot-sg-extension.md

### VS Code Extension

> #### Current Known Limitations

Reviewer: ```suggestion
#### Unsupported features by Sourcegraph version
```

> Here is a list of the known limitation for the VS Code Extension, where some will be addressed in future versions:

Reviewer: ```suggestion
Here is a list of known limitations to the VS Code Extension that we are looking into for future releases:
```
